packages/tabs/src/Tab.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export interface TabProps {
+  /**
+   * Content that will appear as the title in the Tab list.
+   */
+  name: string | React.ReactNode;
```

Comment:
ReactNode includes string. No need to doubly include it.

Reply:
👍